Change 'mobile apps' to just 'apps' (#3452)
dart-lang/site-www: src/multiplatform-apps.md

---

> title: Multi-platform app development
> short-title: Multi-platform apps

Reviewer: Do we really need _development_ in the title? I think it's good to match the sidenav unless there's a compelling reason not to.

```suggestion
title: Multi-platform apps
```

description: Use the Flutter framework to build beautiful native apps for many platforms and operating systems from a single codebase.
toc: false
---

> We recommend the [Flutter framework][] for developing multi-platform native apps for mobile (iOS & Android), desktop (Windows, Linux, and macOS), and the web.

Reviewer 2: ```suggestion
We recommend the [Flutter framework][] for developing multi-platform native apps
for mobile (iOS & Android), desktop (Windows, Linux, and macOS), and the web.
```

Reviewer: & -> and

(We only use `&` in space-constrained places like the short title and sidenav.)

> Flutter is powered by the [Dart platform](/overview#platform), using the
> [Dart VM](/overview#platform) for its instant _hot reload_ developer cycle and
> the [Dart AOT](/overview#native-platform) native code &
> [Dart JS](/overview#web-platform) web compilersfor creating fast production code
> for any platform.

Reviewer: ```suggestion
Flutter is powered by the [Dart platform](/overview#platform).
The [Dart VM](/overview#platform) provides an instant _hot reload_ developer cycle.
The Dart compilers —
an [ahead-of-time (AOT) compiler for native code](/overview#native-platform)
and a [Dart-to-JavaScript compiler](/overview#web-platform) for web code —
create fast production code for any platform.
```
